tests/__mocks__/probot-scheduler.js

```diff
@@ -0,0 +1,2 @@
+"use strict";
+module.exports = () => { };
```

Comment:
I'm not familiar with jest mocks. Could you clarify what this is doing?

Reply:
so you can mock any dependency by putting a file inside `__mocks__` folder. So here i just mocked `probot-scheduler` and made the main function as just a dummy as in our case we dont care.